### PR TITLE
Add Promscale scalability and HA docs

### PR DESCRIPTION
**Pull request by ramonguiu**
# Description

We are moving docs from the Promscale source code repos to docs.timescale.com

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

### CODE REVIEW

**Review by Loquacity — APPROVED** [review]

**Comment by Loquacity** [comment]
@VineethReddy02 @ramonguiu Ready for your review.

**Comment by Loquacity** [comment]
@charislam Great edits, thanks! 💖

File: promscale/scale-ha/multi-node.md [hunk 1]
```diff
@@ -22,3 +22,5 @@ SELECT add_prom_node('example_node_name');
 Make sure that you run the `add_prom_node` command as the same database user as
 the one writing data from Promscale.
 </highlight>
+
+[db-multinode]: https://docs.timescale.com/latest/getting-started/setup-multi-node-basic
```
Comment (ramonguiu) [inline promscale/scale-ha/multi-node.md:26]: @Loquacity should this url be an absolute reference to the prod documentation or use a relative path?

File: promscale/scale-ha/prometheus-multi-tenancy.md [hunk 2]
```diff
@@ -19,18 +19,20 @@ Multi-tenancy features:
...
+from a restricted list of tenants, you can list the tenant names with
+`-multi-tenancy-valid-tenants=<tenant_names_seperated_by_commas>`.
+
+By default, when multi-tenancy is enabled, data is rejected if it does not include a tenant name. If you want to accept data without  tenant information,
 set the `-multi-tenancy-allow-non-tenants` flag.
 
-Remember to set the appropriate CLI flags on all of your Promscale instances.
+<highlight type="note">
+When you set a CLI flag on a Promscale instance, remember to set it on all of
+your other Promscale instances as well, if you need them to have the asme
```
Comment (ramonguiu) [inline promscale/scale-ha/prometheus-multi-tenancy.md]: asme -> same

**Review by ramonguiu — COMMENTED** [review]
Looks good. I just made a couple of minor comments / suggestions.

File: promscale/scale-ha/multi-node.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+environment.
+
+To start, set up a multi-node TimescaleDB cluster, following the instructions in
+the [TimescaleDB multi-node section][db-multinode]. When your multi-node cluster
+is running, you can connect Promscale to the access node of the cluster.
+Promscale then creates the tables, objects, and roles that it needs. You can
+connect to the access node to  query Promscale SQL data.
+
+## Expanding the cluster
+When you have a TimescaleDB multi-nopde cluster that is being written to by
```
Comment (Loquacity) [inline promscale/scale-ha/multi-node.md:12]:
```suggestion
When you have a TimescaleDB multi-node cluster that is being written to by
```

File: promscale/scale-ha/multi-node.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Promscale, you can add nodes to the cluster using the `add_data_node()`
+function. This command adds the data node and uses the
```
Comment (Loquacity) [inline promscale/scale-ha/multi-node.md:14]:
```suggestion
function. These commands add the data node and uses the
```

File: promscale/scale-ha/multi-node.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+SELECT add_data_node('example_node_name', host => 'example_host_address')
+SELECT add_prom_node('example_node_name');
+```
+
+<highlight type="note">
+Make sure that you run the `add_prom_node` command as the same database user as
+the one writing data from Promscale.
+</highlight>
+
+[db-multinode]: https://docs.timescale.com/latest/getting-started/setup-multi-node-basic
```
Comment (Loquacity) [inline promscale/scale-ha/multi-node.md:26]:
```suggestion
[db-multinode]: timescaledb/:currentVersion:/getting-started/setup-multi-node-basic/
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: promscale/scale-ha/high-availability.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+```bash
+FIXME
+promscale --high-availability
+```
+
+Promscale automatically elects a single replica in each cluster as the current
+leader. Only data sent from that replica is ingested. If that leader replica
+stops sending data, then a new replica is elected as the leader.
+
+## Leader election using pg_advisory_lock
```
Comment (VineethReddy02) [inline promscale/scale-ha/high-availability.md:64]: @ramonguiu As this advisory lock-based HA system is deprecated in Promscale (in fact removed in `0.8.0`). We should no longer add it into our docs.

Note from `0.8.0` release notes: 

The long-deprecated advisory-lock based leader election feature has been removed. Please use [label-based](https://github.com/timescale/promscale/blob/master/docs/high-availability/prometheus-HA.md#prometheus-leader-election-via-external-labels) leader election instead.

Comment (ramonguiu) [reply promscale/scale-ha/high-availability.md:64]: @VineethReddy02 Makes sense. Could you remove that section?

Comment (VineethReddy02) [reply promscale/scale-ha/high-availability.md:64]: sure

**Review by VineethReddy02 — APPROVED** [review]
LGTM!

**Review by Loquacity — APPROVED** [review]

**Pull request merged**